### ISSUE TITLE
Class function autocomplete

### ISSUE THREAD

**Issue by LinusU** [vknabel/vscode-swift-development-environment#9]
When accepting autocomplete for an override function, the `func` keyword gets added twice, resulting in broken code.

e.g. with this code:

```swift
class Foobar: UIViewController {
  func vie
}
```

and having the cursor after `func vie` and choosing `viewDidAppear(_ animated: Bool)` results in the following code:

```swift
class Foobar: UIViewController {
    func override func viewDidAppear(_ animated: Bool) {
    
    }
}
```

**Comment by vknabel** [comment]
@LinusU Thanks for your feedback. I had a deeper look on that, but it does not seem to be fixed as easy as initially thought.

**Details:** According to [SourceKit/docs/Procol#code-completion](https://github.com/apple/swift/blob/master/tools/SourceKit/docs/Protocol.md#code-completion), I just need to pass SourceKit's `key.num_bytes_to_erase` to `vscode-languageserver`'s `CompletionItem.additionalTextEdits` as `TextEdit.del`, but in reality `key.num_bytes_to_erase` is always `0`. Maybe I misunderstood something?

**Comment by vknabel** [comment]
@rudedogg Probably yes. Can you provide some example code where it happened to you? Are you a Mac or a Linux user?

**Comment by LinusU** [comment]
@vknabel is this fixed? In which version? I would love to test it out ☺️

**Comment by vknabel** [comment]
@LinusU no sorry. I accidentially Hit the wrong Button 🙈

**Comment by LinusU** [comment]
Hehe, I almost guessed that 😄 no problem!

**Comment by rudedogg** [comment]
Hey @vknabel, can you try this:

![screen shot 2018-03-17 at 8 06 49 pm](https://user-images.githubusercontent.com/2127629/37561842-efb90650-2a1e-11e8-9197-242433793ef1.png)
 
[Press Enter for the completion]

![screen shot 2018-03-17 at 8 07 24 pm](https://user-images.githubusercontent.com/2127629/37561843-f38bffc6-2a1e-11e8-96be-b52bffc44460.png)

Let me know if you can reproduce, if not I'll find try and replicate it on the GitHawk iOS project or something.

**Comment by vknabel** [comment]
@LinusU here is another build. It should fix your `func` issue.
@rudedogg I still couldn't reproduce it and I guess this may be related to a different setup, because @renanyoy did have this problem a while back, too. I included a workaround that hopefully fixes your `..` issue, too.

[vscode-swift-development-environment-2.3.0.vsix.zip](https://github.com/vknabel/vscode-swift-development-environment/files/1822622/vscode-swift-development-environment-2.3.0.vsix.zip) also includes #10 and #11.

**Comment by rudedogg** [comment]
Sorry for the really late reply. VS Code was laggy for some reason on my Hackintosh, but new graphics drivers were released so it's running a lot better.

I tried the 2.3.0 VSIX release you linked, and it fixes the double dot`..` issue when accepting a completion candidate 🎉 !

**Comment by vknabel** [comment]
@rudedogg well in that case: I just released 2.3.0.
